crypto/doc/nrf_cc310_mbedcrypto.rst

```diff
@@ -138,7 +132,7 @@ The following API must be used to change the default `calloc`/`free` function:
    This API must be called prior to calling :c:func:`mbedtls_platform_setup`.
 
 .. note::
-   The library will default to use clib calloc/free functions if the :c:func:`mbedtls_platform_set_calloc_free` is not used.
+   The library will otherwise default to use clib calloc/free functions.
```

Comment:
I suggest deleting this `.. note::` box, and combine the sentence on line 135 with the previous one.

Reply:
- And to change the sentence on line 135 to: "Otherwise, the library will default to use clib/calloc/free functions."
- Also, formatting of 'clib/calloc/free' needs to be fixed? If they are 3 separate functions, I suggest to rewrite to , "Otherwise, the library will default to use the `clib`, `calloc`, or `free` functions."

Reply:
They are not three seperate functions. The library clib provides calloc and free.